### ISSUE TITLE
Visually hidden text is not hidden from screen readers

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#3176]
## Description  
There are many instances of visually hidden text that is not hidden from screen readers on https://clarity.design/documentation/color. This text is styled with the CSS class "copied-indication" and each instance contains the word "Copied!". This text is visually hidden with "opacity: 0;"  but screen readers can still access it. Screen reader users are not getting the benefit of having this text hidden from them as intended for sighted users.

See the relevant HTML/CSS below.

`<span _ngcontent-c4="" class="copied-indication">Copied!</span>`

```
li.card-swatch[_ngcontent-c4] .card-swatch-content[_ngcontent-c4] .rgb-value[_ngcontent-c4] .copied-indication[_ngcontent-c4] {
    display: inline-block;
    -webkit-transform: translateX(-24px);
    transform: translateX(-24px);
    opacity: 0;
    transition: all .2s ease-in-out;
}
```

## Location   
https://clarity.design/documentation/color
  
## Priority     
Medium

## Recommended fix     
If the instances of the word "Copied!" are necessary content than they should be accessible to all users and should not be visibly hidden with "opacity: 0;". If they need to be hidden from all users then they should be removed from the DOM. If this is not possible, aria-hidden="true" should be used on all of them as seen below.

`<span _ngcontent-c4="" aria-hidden="true"  class="copied-indication">Copied!</span>`

 
## LOE  
Medium

**Comment by AlexMarcuBytex** [comment]
hello @chlane ! Are there any conclusions on this issue?
It is possible to eliminate those spans from the DOM, but i think you cant control the transitions by using this approach. However, you can change the visibility property which can be modified through transitions. I find that elements with `visibility:hidden`, will not be seen nor heard by screen readers. I can implement either this alernative or yours, by adding aria-hidden on true;

**Comment by AlexMarcuBytex** [comment]
@chlane im gonna pr with this change today. please assign this issue to me

**Comment by hippee-lee** [comment]
Thanks Alex. We will leave feedback on the PR.
